src/projection.js

```diff
@@ -167,8 +167,10 @@ function scaleProjection(createProjection, kx, ky) {
       if (precision != null) projection.precision?.(precision);
       if (rotate != null) projection.rotate?.(rotate);
       if (typeof clip === "number") projection.clipAngle?.(clip);
-      projection.scale(Math.min(width / kx, height / ky));
-      projection.translate([width / 2, height / 2]);
+      if (width && height) {
```

Comment:
Should this check for nullish instead of zero?

Reply:
good catch! I was thinking aspectRatio, but this part returns the actual projection, so it should respect a collapsed (degenerate) projection.

I fixed it and added a test case (without the fix, the test case displays a map in the default scale, instead of a 0-height map…).

Note that the degenerate case is still slightly problematic, as its path’s `d` property is an invalid string `"M NaN,NaN…"` instead of being null. (A projection with a strictly negative scale returns null, so the "issue" is only for scale 0.)